src/main/java/pl/edu/agh/awi/persistence/model/AbstractForecast.java

```diff
@@ -0,0 +1,138 @@
+package pl.edu.agh.awi.persistence.model;
+
+import org.springframework.data.neo4j.annotation.GraphId;
+
+import java.util.Date;
+
+
+public abstract class AbstractForecast {
```

Comment:
Being exact (and pain in the ass): METAR is not Forecast but rather actual state of Airport.